public/resources/core/js/api/party/individual.service.js

```diff
@@ -0,0 +1,100 @@
+/**
```

Comment:
Why have you used a new folder to this file? Cannot you use the `services` folder you have created for the rest of APIs?